src/components/Icon.tsx

```diff
@@ -11,24 +11,28 @@ export interface IconProps extends AriaAttributes, DOMProps {
   inc?: number;
   /** Styles overrides */
   xss?: Xss<Margin | "visibility">;
+  helperText?: React.ReactNode;
 }
 
 export const Icon = React.memo((props: IconProps) => {
-  const { icon, inc = 3, color = "currentColor", xss, ...other } = props;
+  const { icon, inc = 3, color = "currentColor", xss, helperText, ...other } = props;
   const size = increment(inc);
   return (
-    <svg
-      aria-hidden={true}
-      width={size}
-      height={size}
-      viewBox="0 0 24 24"
-      xmlns="http://www.w3.org/2000/svg"
-      css={{ "path, rect": Css.fill(color).$, ...xss }}
-      data-icon={icon}
-      {...other}
-    >
-      {Icons[icon]}
-    </svg>
+    <div css={Css.df.gap1.color(color).$}>
```

Comment:
thought: I'm curious if the extra div is going to throw off existing usages that don't expect the root element to be a flex container instead of an svg.

Granted, we try to drive home the "callers can't couple themselves to DOM internals of a component", but sometimes especially for the "oh hey the child is flex" change then I _think_ there can be browser/layout-ism changes? @bdow ?

I'm good with merging and just assuming/hoping it will be fine; just thinking that we might have to conditionally wrap the div if `helperText` is set, as we've done that in a few other places for similar sort of reasons.

Reply:
I agree. I think I'd rather the Icon button stay as returning only the svg. If a component needs to add a label next to it or decorate it in some fashion, then I think that component should import the Icon element as an svg and style any containers needed for the Icon themselves. Icon is one of our components that is dead simple, and I really like that about it.

Reply:
The only different Chromatic noted was this, and it really wasn't a difference: https://www.chromatic.com/test?appId=60466f7d43c37c0021788867&id=629a76013d6416003afee83e

But I'm not tethered to this approach.

Reply:
Great, let's move this wrapping element into the `AutoSaveIndicator` component. And maybe we wrap it with a `display: inline-flex;` instead. This will help prevent the tooltip from getting placed far away from the component.
<img width="1555" alt="Screen Shot 2022-06-06 at 9 27 45 AM" src="https://user-images.githubusercontent.com/1143861/172170082-d297b4ad-16c2-4d67-98be-43b1ca1e9806.png">

Reply:
Man, I was trying so many things to get that tooltip to be placed correctly. Thanks 😃